tools/check_tidy.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+# Copyright 2015-present Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+
+from argparse import ArgumentParser
+from difflib import unified_diff
+#from check_license import CheckLicenser
```

Comment:
is there a plan to support licenses check or this is leftover code?

Reply:
I don't have any plan. I'll remove it if I want